masschallenge/django-accelerator
# [AC-9098] 0. Create PartnerLabel model

## [AC-9098](https://masschallenge.atlassian.net/browse/AC-9098)

**Changes introduced**
- Added PartnerLabel model

**How to test**
- Ensure that all the changes stated in [AC-9098](https://masschallenge.atlassian.net/browse/AC-9098) are met.
